server/routes/overview/overviewView.ts

```diff
@@ -1,23 +1,56 @@
-import type { ActionPlanReviews, FunctionalSkills, InPrisonCourseRecords, PrisonerSummary } from 'viewModels'
+import type {
+  ActionPlanReviews,
+  Assessment,
+  FunctionalSkills,
+  InPrisonCourse,
+  InPrisonCourseRecords,
+  PrisonerGoals,
+  PrisonerSummary,
+} from 'viewModels'
+import { startOfToday, sub } from 'date-fns'
+import type { InductionDto } from 'inductionDto'
 import dateComparator from '../dateComparator'
 
 type RenderArgs = {
   tab: string
   prisonerSummary: PrisonerSummary
-  functionalSkills: FunctionalSkills
-  inPrisonCourses: InPrisonCourseRecords
-  actionPlanReviews: ActionPlanReviews
-  isPostInduction: boolean
-  lastUpdatedBy: string | null
-  lastUpdatedDate: Date | null
-  lastUpdatedAtPrisonName: string | null
-  noGoals: boolean
-  goalCounts: { activeCount: number; archivedCount: number; completedCount: number }
-  hasWithdrawnOrInProgressCourses: boolean
-  hasCoursesCompletedMoreThan12MonthsAgo: boolean
-  problemRetrievingData: boolean
-  showServiceOnboardingBanner: boolean
-  actionPlanReviewsCount: string
+  functionalSkills: {
```

Comment:
There's a lot of change here in the view class, but basically this `RenderArgs` type is the crux of it. This is the spec/definition of the rendered properties in the view 👍 , and all the logic for fulfilling this contract is within this class 👍